cts/cli/regression.error_codes.exp

```diff
@@ -101,23 +101,23 @@ pcmk_err_no_quorum - Operation requires quorum
 =#=#=#= End test: List legacy return codes (spot check) (with names) (XML) - OK (0) =#=#=#=
 * Passed: crm_error             - List legacy return codes (spot check) (with names) (XML)
 =#=#=#= Begin test: Get unknown Pacemaker return code =#=#=#=
-Error
+Unknown error 10000
```

Comment:
This change and the other similar ones break at least FreeBSD in CI. FreeBSD adds a colon.
```
< Unknown error 10000
---
> Unknown error: 10000
109c109
<   <result-code code="10000" description="Unknown error 10000"/>
---
>   <result-code code="10000" description="Unknown error: 10000"/>
115c115
< Unknown - Unknown error 10000
---
> Unknown - Unknown error: 10000
120c120
<   <result-code code="10000" name="Unknown" description="Unknown error 10000"/>
---
>   <result-code code="10000" name="Unknown" description="Unknown error: 10000"/>
548,551d547
```